ethstorage/prover/zk_prover.go

```diff
@@ -4,69 +4,46 @@
 package prover
 
 import (
-	"encoding/json"
-	"errors"
 	"fmt"
 	"math/big"
 	"os"
 	"os/exec"
 	"path/filepath"
-	"strings"
 	"time"
 
-	"github.com/consensys/gnark-crypto/ecc/bn254/fr"
 	"github.com/crate-crypto/go-proto-danksharding-crypto/eth"
-	"github.com/ethereum/go-ethereum/accounts/abi"
 	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/log"
 )
 
 const (
-	snarkLibDir      = "snarkjs"
 	snarkBuildDir    = "snarkbuild"
 	witnessGenerator = "generate_witness.js"
+	witnessName      = "witness_blob_poseidon.wtns"
 	inputName        = "input_blob_poseidon.json"
-	wasmName         = "blob_poseidon.wasm"
-	wasmName2        = "blob_poseidon2.wasm"
-	wtnsName         = "witness_blob_poseidon.wtns"
 	proofName        = "proof_blob_poseidon.json"
 	publicName       = "public_blob_poseidon.json"
 )
 
 type ZKProver struct {
```

Comment:
Is it using snarkjs? need to change to zk_prover_js.go and ZKProver?

Reply:
I think it's OK to keep the original name because it is the default implementation, plus it's more convenient to compare the changes.